bottles/frontend/bottles.py

```diff
@@ -17,6 +17,8 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+import os
+os.environ.pop('GTK_THEME')
```

Comment:
Does it have to be here? I would prefer to have it below the imports

Reply:
No it probably does not have to be there, but doing this as early as possible would ensure that gtk is not initialized before removing the environment variable